### DISCUSSION TITLE
Enabling Blog and Image_Text_button Section for Horizon

### DISCUSSION THREAD

**Discussion by flonUK** [swellstores/community#480] [Help] [answered]
Hi, does anyone know how to enable the [blog](https://swell-horizon-demo.vercel.app/blog) section through swell admin and I can't seem to find this section in the screenshot below either <img width="1232" alt="Screenshot 2024-01-18 at 20 02 04" src="https://github.com/swellstores/community/assets/156616226/ede844ea-d743-476e-b3bb-cb02bba91f46">
 Thanks

**Comment by logeshswell** [comment, marked as answer]
Inside the storefront editor, create a new page **Blog** and create content inside it.
<img width="1512" alt="Screenshot 2024-01-23 at 1 59 42 PM" src="https://github.com/swellstores/community/assets/100766131/ce9efc8f-1148-4ef3-b9f5-3bfbfae7f525">

To enable the Storefront editor for Horizon theme in Swell admin dashboard, set env `NEXT_PUBLIC_SWELL_EDITOR=true` as per the [README instructions](https://github.com/swellstores/horizon?tab=readme-ov-file#setting-up-your-environment).

**Reply by flonUK** [reply]
thanks for the info on the blog, is there a fix for the multiple panel yet? thanks
